### ISSUE TITLE
Create T1w and T2w templates using the same target template space

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/pediatric-template#10]
To make sure both templates have the same size. 

Context: https://github.com/spinalcordtoolbox/pediatric-template/pull/9

**Comment by rohanbanerjee** [comment]
Both the T1w and the T2w templates are in the same space now. Following the suggestion from this: https://github.com/spinalcordtoolbox/pediatric-template/pull/9#issuecomment-1743983571 , the below are the steps I followed in order to get them in the same space:

1. Using the T1w images and derivatives, I used the script `preprocess_normalize.py` just as mentioned in the steps of the pipeline of template generation.
2. Once I had the template space, template masks etc from the T1w images, I updated my config file for T2w images.
3. This time for the T2w template, in the `preprocess_normalize.py` pipeline, I commented out the `generate_initial_template_space` method/step. This means that while running the pipeline this time, a new `template space` with the different dimensions of the T2w won't be generated and it would use the `template space` which was generated using the T1w images.
4. As the next step in the pipeline was straightening, it straightens the images using the T1w template space and therefore the T1w and the T2w come to the same space.

As mentioned here: https://github.com/spinalcordtoolbox/pediatric-template/pull/9#issuecomment-1742664210, previously the sizes of the the T1w and the T2w templates were different but now both of them have the same size -- i.e. **201 x 201 x 688**
